uisdk/src/main/res/layout/uisdk_activity_booking_content.xml

```diff
@@ -39,7 +39,8 @@
         android:layout_height="@dimen/kh_uisdk_collapsible_pane_expanded_height"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent"/>
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:expandedListPercentageOfScreen="70"/>
```

Comment:
can we use kh_uisdk_query_list_view_default_screen_percentage instead of hardcoded number?